compiler/include/symbol.h

```diff
@@ -462,6 +462,8 @@ class ModuleSymbol : public Symbol {
   void                 moduleUseAdd(ModuleSymbol* module);
   void                 moduleUseRemove(ModuleSymbol* module);
 
+  bool                 isVisible(BaseAST* scope);
```

Comment:
The answer should not change if the same scope is sent in multiple times, I believe, so that would argue for the method being declared const (if my understanding is correct).  I am a little worried I don't understand the full ramifications of this decision, though, and so would probably not do it for that reason.